src/wizards/tokenSwapDealWizard/components/daoStageContent/daoStageContent.ts

```diff
@@ -1,7 +1,32 @@
 import { IDAO } from "entities/DealRegistrationTokenSwap";
 import { availableSocialMedias } from "../../dealWizardTypes";
-import { bindable, inject } from "aurelia";
+import { bindable, inject, observable } from "aurelia";
 import { IValidationController } from "@aurelia/validation-html";
+import { FirestoreService } from "services/FirestoreService";
+import { IDAOsData } from "services";
+import { ethers } from "ethers";
+import { Utils } from "services/utils";
+import "cl-webcomp-poc";
```

Comment:
But now this means, we import _all_ the cl-webcomp-poc ?
Like there are at least 2 other webcomp in that lib